spot_driver/spot_driver/spot_ros2.py

```diff
@@ -1466,245 +1470,216 @@ def handle_max_vel(self, request: SetVelocity.Request, response: SetVelocity.Res
+                or fb.arm_cartesian_feedback.status.value == fb.arm_cartesian_feedback.status.STATUS_TRAJECTORY_STALLED
             ):
-                if (
-                    full_body_feedback.feedback.safe_power_off_feedback.status.value
-                    == full_body_feedback.feedback.safe_power_off_feedback.status.STATUS_POWERED_OFF
-                ):
-                    return GoalResponse.SUCCESS
-                else:
-                    return GoalResponse.IN_PROGRESS
-            elif (
-                full_body_feedback.feedback.feedback_choice
-                == full_body_feedback.feedback.FEEDBACK_BATTERY_CHANGE_POSE_FEEDBACK_SET
+                return GoalResponse.FAILED
+            if fb.arm_cartesian_feedback.status.value != fb.arm_cartesian_feedback.status.STATUS_TRAJECTORY_COMPLETE:
+                return GoalResponse.IN_PROGRESS
+        elif choice == fb.FEEDBACK_ARM_JOINT_MOVE_FEEDBACK_SET:
+            if fb.arm_joint_move_feedback.status.value == fb.arm_joint_move_feedback.status.STATUS_STALLED:
+                return GoalResponse.FAILED
+            if fb.arm_joint_move_feedback.status.value != fb.arm_joint_move_feedback.status.STATUS_COMPLETE:
+                return GoalResponse.IN_PROGRESS
+        elif choice == fb.FEEDBACK_NAMED_ARM_POSITION_FEEDBACK_SET:
+            if (
+                fb.named_arm_position_feedback.status.value
+                == fb.named_arm_position_feedback.status.STATUS_STALLED_HOLDING_ITEM
             ):
-                if (
-                    full_body_feedback.feedback.battery_change_pose_feedback.status.value
-                    == full_body_feedback.feedback.battery_change_pose_feedback.status.STATUS_COMPLETED
-                ):
-                    return GoalResponse.SUCCESS
-                if (
-                    full_body_feedback.feedback.battery_change_pose_feedback.status.value
-                    == full_body_feedback.feedback.battery_change_pose_feedback.status.STATUS_FAILED
-                ):
-                    return GoalResponse.FAILED
+                return GoalResponse.FAILED
+            if fb.named_arm_position_feedback.status.value != fb.named_arm_position_feedback.status.STATUS_COMPLETE:
+                return GoalResponse.IN_PROGRESS
+        elif choice == fb.FEEDBACK_ARM_VELOCITY_FEEDBACK_SET:
+            self.get_logger().warn("WARNING: ArmVelocityCommand provides no feedback")
+        elif choice == fb.FEEDBACK_ARM_GAZE_FEEDBACK_SET:
+            if fb.arm_gaze_feedback.status.value == fb.arm_gaze_feedback.status.STATUS_TOOL_TRAJECTORY_STALLED:
+                return GoalResponse.FAILED
+            if fb.arm_gaze_feedback.status.value != fb.arm_gaze_feedback.status.STATUS_TRAJECTORY_COMPLETE:
+                return GoalResponse.IN_PROGRESS
+        elif choice == fb.FEEDBACK_ARM_STOP_FEEDBACK_SET:
+            self.get_logger().warn("WARNING: Stop command provides no feedback")
```

Comment:
Spam?

Reply:
I believe in these cases where no feedback is provided, `_robot_command_goal_complete` will return `GoalResponse.SUCCESS` and proceed to the next command. So it shouldn't spam unless we are receiving endless stop commands

Reply:
And the same should apply to the other commands that don't provide feedback. I could just make these comments